template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/profilermanager/ProfilerMetricsRoleConfigProvider.java

```diff
@@ -17,32 +17,35 @@
 @Component
 public class ProfilerMetricsRoleConfigProvider extends AbstractRdsRoleConfigProvider {
 
-    @VisibleForTesting
-    static final String PROFILER_METRICS_DATABASE_HOST = "profiler_metrics_database_host";
+    public static final String PROFILER_METRICS_DATABASE_HOST = "profiler_metrics_database_host";
 
-    @VisibleForTesting
-    static final String PROFILER_METRICS_DATABASE_NAME = "profiler_metrics_database_name";
+    public static final String PROFILER_METRICS_DATABASE_NAME = "profiler_metrics_database_name";
 
-    @VisibleForTesting
-    static final String PROFILER_METRICS_DATABASE_TYPE = "profiler_metrics_database_type";
+    public static final String PROFILER_METRICS_DATABASE_TYPE = "profiler_metrics_database_type";
 
-    @VisibleForTesting
-    static final String PROFILER_METRICS_DATABASE_USER = "profiler_metrics_database_user";
+    public static final String PROFILER_METRICS_DATABASE_USER = "profiler_metrics_database_user";
 
-    @VisibleForTesting
-    static final String PROFILER_METRICS_DATABASE_PASSWORD = "profiler_metrics_database_password";
+    public static final String PROFILER_METRICS_DATABASE_PASSWORD = "profiler_metrics_database_password";
+
+    public static final String PROFILER_METRICS_DATABASE_JDBC_URL_OVERRIDE = "profiler_metrics_database_jdbc_url_override";
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
+        List<ApiClusterTemplateConfig> configs = new ArrayList<>();
         if (ProfilerManagerRoles.PROFILER_METRICS_AGENT.equals(roleType)) {
             RdsView profilerManagerRdsView = getRdsView(source);
-            return List.of(config(PROFILER_METRICS_DATABASE_HOST, profilerManagerRdsView.getHost()),
-                    config(PROFILER_METRICS_DATABASE_NAME, profilerManagerRdsView.getDatabaseName()),
-                    config(PROFILER_METRICS_DATABASE_TYPE, ConfigUtils.getDbTypePostgres(profilerManagerRdsView, ProfilerManagerRoles.PROFILER_MANAGER)),
-                    config(PROFILER_METRICS_DATABASE_USER, profilerManagerRdsView.getConnectionUserName()),
-                    config(PROFILER_METRICS_DATABASE_PASSWORD, profilerManagerRdsView.getConnectionPassword()));
+            configs.add(config(PROFILER_METRICS_DATABASE_USER, profilerManagerRdsView.getConnectionUserName()));
+            configs.add(config(PROFILER_METRICS_DATABASE_PASSWORD, profilerManagerRdsView.getConnectionPassword()));
+            configs.add(config(PROFILER_METRICS_DATABASE_HOST, profilerManagerRdsView.getHost()));
+            if (profilerManagerRdsView.isUseSsl()) {
```

Comment:
Not a big deal but it would perhaps be sufficient to set `PROFILER_METRICS_DATABASE_HOST` in the `else` branch, i.e. when SSL is not enabled.